cnf-certification-test/lifecycle/suite.go

```diff
@@ -224,7 +224,9 @@ func testContainersPostStart(env *provider.TestEnvironment) {
 			tnf.ClaimFilePrintf("%s does not have postStart defined", cut)
 			nonCompliantObjects = append(nonCompliantObjects, testhelper.NewContainerReportObject(cut.Namespace, cut.Podname, cut.Name, "Container does not have postStart defined", false))
 		} else {
-			compliantObjects = append(compliantObjects, testhelper.NewContainerReportObject(cut.Namespace, cut.Podname, cut.Name, "Container has postStart defined", true))
+			compliantObjects = append(compliantObjects, testhelper.NewContainerReportObject(cut.Namespace, cut.Podname, cut.Name, "Container has postStart defined,"+
```

Comment:
Was the intent to add this to the main branch or ginkgo_removal?

Reply:
its for both so i need to recreate on ginkgo_removal?

Reply:
https://github.com/test-network-function/cnf-certification-test/pull/1727